rootfs/etc/nginx/lua/plugins/statsd_monitor/statsd.lua

```diff
@@ -160,7 +160,7 @@ end
 _M.config = {
   host = os.getenv("STATSD_HOST"),
   port = os.getenv("STATSD_PORT"),
-  sampling_rate = 1.0,
+  sampling_rate = 0.1,
```

Comment:
the new value is in alignment with https://github.com/Shopify/nginx-routing-modules/blob/eb70859303769666844ae60ccaf0443efb897f5a/config/statsd.json#L11